src/main/java/ui/Printer.java

```diff
@@ -300,4 +302,75 @@ public void printColors() {
+            + NEWLINE + "Enter \"/help\" to see the details of all commands");
+    }
+
+    private void printRepeatedHelpMessagePartOne(String commandType) {
+        print("The " + commandType + " does not work alone."
+            + NEWLINE + "We need some necessary information from you before we could add your cheat sheet."
+            + NEWLINE + "Don't worry, you do not have to enter the command again."
+            + NEWLINE + "We would prompt you to enter the necessary information, one by one.");
+    }
+
+    private void printRepeatedHelpMessagePartTwo() {
+        print("You also could enter the full command next time if you do not wish to be prompted to enter fields "
+            + "one by one" + NEWLINE + "\"/help\" to see more details on the structure of full command."
+            + NEWLINE + "To stop messages like this from appearing again, input this command: \"/set /m remove\"");
+    }
+
+    public void printAddCommandHelpMessage() {
+        printRepeatedHelpMessagePartOne("ADD");
+        print("NAME is necessary while SUBJECT isn't. <<enter>> to skip field."
+            + NEWLINE + "We will automatically assign \"Unsorted\" as subject if no subject is provided." + NEWLINE);
+        printRepeatedHelpMessagePartTwo();
+    }
+
+    public void printClearCommandHelpMessage() {
+        print("The CLEAR command clears ALL cheat sheets except for those that are preloaded."
+            + NEWLINE + "Please consider twice before proceeding");
+    }
+
+    public void printDeleteCommandHelpMessage() {
+        print("The DELETE command deletes a cheatsheet you specifies. Please consider twice before proceeding");
+        printRepeatedHelpMessagePartOne("DELETE");
+        print("We need you to either input NAME or INDEX. <<enter>> to skip field.");
+        printRepeatedHelpMessagePartTwo();
+    }
+
+    public void printEditCommandHelpMessage() {
+        print("The EDIT command edits an existing cheat sheet.");
+        printRepeatedHelpMessagePartOne("EDIT");
+        print("We need you to input either NAME or INDEX. <<enter>> to skip field.");
+        printRepeatedHelpMessagePartTwo();
+    }
+
+    public void printFindCommandHelpMessage() {
+        print("The EDIT command finds an existing cheat sheet based on information provided.");
+        printRepeatedHelpMessagePartOne("FIND");
+        print("We need you to input NAME, SUBJECT, or KEYWORD in cheatsheet or any combinations of these fields."
+            + "<<enter>> to skip field.");
+        printRepeatedHelpMessagePartTwo();
+    }
+
+    public void printListCommandHelpMessage() {
+        print("The LIST command lists all current existing cheat sheets."
+            + NEWLINE + "You could also sort the cheat sheets according to name or subject alphabetical order"
+            + NEWLINE + "To stop messages like from appearing again, input this command: \"/set /m remove\"");

```

Comment:
Perhaps you could seperate each addition on a separate line for easier readibbility